conda-forge.yml

```
@@ -1,4 +1,10 @@
+azure:
+  free_disk_space: true
+  settings_linux:
+    swapfile_size: 10GiB
 build_platform:
+  linux_aarch64: linux_aarch64
+  linux_ppc64le: linux_ppc64le
```

Think we can try cross-compilation again with the changes above

Generally cross-compilation would be a better experience if we can get that to work

```suggestion
  linux_aarch64: linux_64
  linux_ppc64le: linux_64
```